tracer/src/Datadog.Trace/AppSec/EventTrackingSdk.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <copyright file="EventTrackingSdk.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.AppSec;
```

Comment:
Should this be an AppSec only API? One debate was that this API works for both APM and ASM, so I wasn't sure it would be in the AppSec namespace ?

Reply:
ASM decided the spans tags would be names `appsec.event. ...` so I believe it's correct to place it in the `Datadog.Trace.AppSec` namespace.

I agree the information could be useful to other products and will be displayed in the APM tag explorer, but it would seem it's primary use is ASM.